samples/snippets/logs/read_job_logs.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+# [START batch_job_logs]
+from typing import Generator, NoReturn
+
+from google.cloud import batch_v1
+from google.cloud import logging
+
+
+def get_job_logs(project_id: str, job: batch_v1.Job) -> Generator[logging.TextEntry, None, None]:
+    log_client = logging.Client(project=project_id)
```

Comment:
Please include this comment for client initialization: https://googlecloudplatform.github.io/samples-style-guide/#clients